public/react/Configure/ConfigurationPanel.tsx

```diff
@@ -17,23 +17,27 @@ export interface ConfigurationPanelProps {
   onBackButtonClicked: () => void;
   onUpdate?: (overrides: Partial<NormalizeIssueConfig>) => void;
   derivedIssuesObservable: CanObservable<Array<{ team: { name: string } }> | undefined>;
+  showSidebarBranding: CanObservable<boolean>;
 }
 
 type TeamName = "global" | (string & {});
 
 const ConfigurationPanel: FC<ConfigurationPanelProps> = ({
   onBackButtonClicked,
   derivedIssuesObservable,
+  showSidebarBranding,
   ...configurationProps
 }) => {
   const jiraFields = useJiraIssueFields();
   const { savedUserAllTeamData } = useAllTeamData(jiraFields);
+  const isShowSidebarBranding = useCanObservable(showSidebarBranding);
```

Comment:
is this an observable? I thought it was just a true or false passed from mainHelper